Source: written by Pierre Lejeune <superheron@gmailcom> for Exherbo
Upstream: Submitted, ticket #245
Reason: Use cinit instead of new
--- tools/python-yasm/yasm.pyx.orig	2010-08-07 18:34:30.000000000 +0200
+++ tools/python-yasm/yasm.pyx	2011-02-13 18:12:56.076551357 +0100
@@ -97,7 +97,7 @@
 #
 cdef class __assoc_data_callback:
     cdef yasm_assoc_data_callback *cb
-    def __new__(self, destroy, print_):
+    def __cinit__(self, destroy, print_):
         self.cb = <yasm_assoc_data_callback *>malloc(sizeof(yasm_assoc_data_callback))
         self.cb.destroy = <void (*) (void *)>PyCObject_AsVoidPtr(destroy)
         #self.cb.print_ = <void (*) (void *, FILE *, int)>PyCObject_AsVoidPtr(print_)
@@ -107,7 +107,7 @@
 
 cdef class Register:
     cdef unsigned long reg
-    def __new__(self, reg):
+    def __cinit__(self, reg):
         self.reg = reg
 
 include "errwarn.pxi"
